package.json

```diff
@@ -17,55 +17,62 @@
   "repository": "https://github.com/ember-cli/ember-page-title",
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "npm-run-all lint:* test:*",
+    "test": "npm-run-all lint test:*",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each",
     "format:hbs": "prettier **/*.hbs --write --parser=glimmer",
     "format:js": "prettier **/*.js --write",
     "release": "standard-version"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.22.1"
+    "ember-cli-babel": "^7.23.1"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
-    "@glimmer/component": "^1.0.2",
-    "@glimmer/tracking": "^1.0.2",
+    "@ember/test-helpers": "^2.2.0",
+    "@glimmer/component": "^1.0.3",
+    "@glimmer/tracking": "^1.0.3",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-auto-import": "^1.6.0",
-    "ember-cli": "~3.21.2",
+    "ember-auto-import": "^1.10.1",
+    "ember-cli": "~3.25.2",
     "ember-cli-code-coverage": "^1.0.0",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-fastboot": "^2.2.3",
     "ember-cli-fastboot-testing": "^0.4.0",
     "ember-cli-github-pages": "^0.2.0",
-    "ember-cli-htmlbars": "^5.3.1",
+    "ember-cli-htmlbars": "^5.3.2",
     "ember-cli-inject-live-reload": "^2.0.2",
     "ember-cli-sass": "^10.0.1",
-    "ember-cli-terser": "^4.0.0",
+    "ember-cli-terser": "^4.0.1",
     "ember-cli-update": "^0.54.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-inline-svg": "^1.0.0",
-    "ember-load-initializers": "^2.1.1",
+    "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.6.0",
+    "ember-qunit": "5.1.1",
     "ember-resolver": "^8.0.2",
-    "ember-source": "~3.21.1",
+    "ember-source": "~3.25.3",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^2.13.0",
+    "ember-template-lint": "^2.18.1",
     "ember-truth-helpers": "^2.1.0",
     "ember-try": "^1.4.0",
-    "eslint": "^7.10.0",
-    "eslint-plugin-ember": "^9.0.0",
+    "eslint": "^7.20.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-ember": "^10.2.0",
     "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.3.1",
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.1.2",
+    "prettier": "^2.2.1",
+    "qunit": "2.13.0",
```

Comment:
Temp pin `ember-qunit` and `qunit`. More info in the related commit and [here on Discord](https://discord.com/channels/480462759797063690/568935504288940056/818524713495429140). Will create a new PR to unpin again when fixed in `qunit` (or we can wait with merging of course until the fix has landed).